.github/workflows/docbuild.yml

```diff
@@ -63,6 +71,31 @@ jobs:
         run: |
           west zephyr-export
 
+      - name: Check added or updated links
+        working-directory: ncs/nrf
+        run: |
+          RE=".. _\`(.*)\`: (.*)"
+          LINKS=$(git diff --unified=0 "${{ github.event.pull_request.base.sha }}..HEAD" | \
```

Comment:
for @nrfconnect/ncs-ci: It looks like adding `-- doc/nrf/links.txt` makes this command fail (without any error message) when `doc/nrf/links.txt` is not edited. Any ideas? Now it's subobtmical as it checks all diff...

Reply:
for me it looks that if/when git diff return nothing (e.g. no changes to links.txt) then grep "^+"    will  raise exit code. So || true at the end should probably fix it.

Reply:
thanks, that was it!